R/cheetah_utils.R

```diff
@@ -7,24 +7,32 @@
 #' @param width Column width.
 #' @param min_width Column minimal width.
 #' @param max_width Column max width.
-#' @param column_type Column type. There are 6 possible options:
+#' @param column_type Column type. By default, the column type is inferred from the data type of the column.
+#' There are 7 possible options:
 #' \itemize{
 #'   \item \code{"text"} for text columns.
```

Comment:
We can use markdown syntax in roxygen2. It's easier than having \itemize ...
Instead it can be:

```r
#' There are 7 possible options:
#' - Option 1
#' - Option 2
```